packages/expo-location/android/build.gradle

```diff
@@ -66,10 +66,11 @@ if (new File(rootProject.projectDir.parentFile, 'pubspec.yaml').exists()) {
 
 dependencies {
   expendency "expo-core"
+  expendency "expo-errors"
   expendency "expo-permissions-interface"
   expendency "expo-task-manager-interface"
 
-  api 'com.google.android.gms:play-services-location:15.0.1'
+  api 'com.google.android.gms:play-services-location:16.0.0'
```

Comment:
Can we safely do it without upgrading other `play-services`?

Reply:
According to https://developers.google.com/android/guides/versioning
> Each Maven dependency matching com.google.android.gms:play-services-* and com.google.firebase:firebase-*, after 15.0.0, is no longer required to have the same version number in order to work correctly at build time and at run time.
> 
> Individual library updates may not be released at the same time - they may be updated and versioned independently. Components may now have a faster cycle for bug fixes and new features.